karma.conf.js

```diff
@@ -53,10 +53,13 @@ module.exports = function(config) {
         // Preprocess matching files before serving them to the browser.
         // https://npmjs.org/browse/keyword/karma-preprocessor
         preprocessors: {
+            '*.js': ['babel'],
+            'src/**/*.js': ['babel'],
+            'platform/**/src/**/*.js': ['babel'],
+            'platform/**/test/**/*.js': ['babel'],
             'src/**/src/**/!(*Spec).js': [ 'coverage' ],
```

Comment:
I think it looks neat enough the way it is. I can't completely remove both `coverage` lines because I want to run Specs and Non-Specs through the ES6 preprocessor. I could merge the `src` paths, but I think this is fine.